packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -1224,230 +1683,141 @@ describe("handleNewBooking", () => {
               // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
               uid: createdBooking.uid!,
               eventTypeId: mockBookingData.eventTypeId,
-              // Rescheduled booking sill stays in pending state
-              status: BookingStatus.PENDING,
+              status: BookingStatus.ACCEPTED,
               location: BookingLocations.CalVideo,
               responses: expect.objectContaining({
                 email: booker.email,
                 name: booker.name,
               }),
-              references: [
-                {
-                  type: appStoreMetadata.dailyvideo.type,
-                  uid: "MOCK_ID",
-                  meetingId: "MOCK_ID",
-                  meetingPassword: "MOCK_PASS",
-                  meetingUrl: "http://mock-dailyvideo.example.com",
-                },
-                {
-                  type: appStoreMetadata.googlecalendar.type,
-                  uid: "MOCK_ID",
-                  meetingId: "MOCK_ID",
-                  meetingPassword: "MOCK_PASSWORD",
-                  meetingUrl: "https://UNUSED_URL",
-                  externalCalendarId: "MOCK_EXTERNAL_CALENDAR_ID",
-                },
-              ],
             },
           });
 
-          expectWorkflowToBeTriggered();
-
-          expectBookingRequestedEmails({
-            booker,
-            organizer,
+          expectSuccessfulRoudRobinReschedulingEmails({
```

Comment:
Expects that roundRobinHost1 receives cancelation email and roundRobinHost2 to receive scheduling email